hack/apidiff.sh

```diff
@@ -144,31 +160,44 @@ run () {
     done
 }
 
-# runWorktree checks out a specific revision, then invokes run there.
-runWorktree () {
-    local out="$1"
-    local worktree="$2"
-    local rev="$3"
+# inWorktree checks out a specific revision, then invokes the given
+# command there.
+#
+# shellcheck disable=SC2317 # "Command appears to be unreachable" - gets called indirectly.
+inWorktree () {
+    local worktree="$1"
+    shift
+    local rev="$1"
+    shift
 
     # Create a copy of the repo with the specific revision checked out.
-    git worktree add -f -d "${worktree}" "${rev}"
-    # Clean up the copy on exit.
-    kube::util::trap_add "git worktree remove -f ${worktree}" EXIT
+    # Might already have been done before.
+    if ! [ -d "${worktree}" ]; then
```

Comment:
can happen that the folder exist but does not have the version we want?

Reply:
No. It gets created in a fresh, empty `KUBE_TEMP`. The first `apidiff` invocation creates it, then later the trial builds can reuse it. Instead of tracking that it was easier to simply check for the directory here.